shoop/core/models/_contacts.py

```diff
@@ -359,7 +359,7 @@ def get_person_contact(user):
         'last_name': getattr(user, 'last_name', ''),
         'email': getattr(user, 'email', ''),
     }
-    return PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
+    return PersonContact.objects.filter(is_active=True).get_or_create(user=user, defaults=defaults)[0]
```

Comment:
So since the in active contacts can't login this happens when the user is still logged in when it is deactivated. Also this will cause dublicate entry database error since the user is onetoone-field.

I think the correct fix here would be logout the user and show message that your account is deactivated.

I think this shouldn't never return None. Also user shouldn't be logged in without actual active contact in request.